### PR TITLE
When uploading a new image, the last image used by the user is deleted from the container.

### PR DESCRIPTION
**Pull request by Oozaru71**
I am not experiencing the "Image upload button when clicked shows two dialogs." issue you mentioned.

### CODE REVIEW

File: Controllers/ImagesController.cs [hunk 1]
```diff
@@ -150,6 +155,24 @@ public async Task<IActionResult> Upload(ICollection<IFormFile> files)
                 return BadRequest(ex.Message);
             }
               
+        }
+        public void DeleteBlob(string deleteFile)
+        {
+             
+            BlobContainerClient containerC = new BlobContainerClient("DefaultEndpointsProtocol=https;AccountName=devstorageale;AccountKey=l2Fo6os7kVCIZZ992p5NavyHar3SjUMQStiV1Ac89NLSJOOTPu6RAlmf9QoRGnoILvPZpYYePFkl+AStmsx10w==;EndpointSuffix=core.windows.net", "muszilla");
```
Comment (Alejandro-HUB) [inline Controllers/ImagesController.cs]: Let's try not to use credentials in the code. Try to add this to app settings or to resources. It is bad practice to have it so easily accesible.

**Review by Alejandro-HUB — CHANGES_REQUESTED** [review]

**Comment by Oozaru71** [comment]
I have fixed the credential issue as well as the upload screens issue.

**Comment by Oozaru71** [comment]
#3 completed

**Review by Alejandro-HUB — APPROVED** [review]

**Comment by Alejandro-HUB** [comment]
Approved, great job Michael. I will get started on making our following tasks.

**Pull request merged**